### ISSUE TITLE
Bump udns version

### ISSUE THREAD

**Issue by mirolm** [stickz/rtorrent#51]
### Is your feature request related to a problem?

The version of udns library is not the latest. Newer one may bring some bugfixes and new features.

### Describe the solution you'd like

I went to the project [official site](https://www.corpit.ru/mjt/udns.html) and there we have version 0.6 from this summer. I'm not sure if this will bring any real benefits for the project but maybe it is worth to look at.

**Comment by stickz** [comment]
Yes, if you'd like to submit a pull request to update the readme we can do that. Otherwise, I'll do it myself but it will take some time. The priority is medium for this task. The bugfixes from this release will have a very minimal impact.

**Comment by mirolm** [comment]
I will leave to you as i do not know if i can do it right.
